src/backend/bs_publish

```diff
@@ -975,6 +975,20 @@ sub createrepo_debian {
   } else {
     rmdir($udebs);
   }
+
+  my $ddebs = "$extrep/debian-debug";
+  mkdir_p($ddebs) unless -d $ddebs;
+  if (qsystem('chdir', $ddebs, 'stdout', 'Packages.new', 'dpkg-scanpackages', '-t', 'ddeb', '-m', '..', '/dev/null')) {
```

Comment:
The debug packages automatically generated by debhelper are actually called foo-dbgsym.deb - see list on http://debug.mirrors.debian.org/debian-debug/pool/main/

Reply:
To my understanding this was only the case for the manually created packages. With a newer compat level, those should be generated automatically with a ddeb ending. From my understanding the general idea of the ddeb files was, to be able to handle the ddeb files separately in case you want to mirror a repository. 
https://wiki.debian.org/AutomaticDebugPackages

Reply:
That's not the case, if you look at the repository i linked you'll see it

Reply:
Ok, looks like it's at the moment only the case for Ubuntu https://wiki.ubuntu.com/Debug%20Symbol%20Packages
For Ubuntu the main repository at http://archive.ubuntu.com/ubuntu/ does not contain any dbgsym/ddeb files, those are located here http://ddebs.ubuntu.com/
It looks like, that Ubuntu seems to differ in the implemented behavior of dh_strip

Reply:
It's not just for the moment, it's never going to be any different. Match for -dbgsym instead and it will work for both.

Reply:
Sure, but this wouldn't be possible with the current implementation of dpkg-scanpackages, since -t defines the used file extension https://git.dpkg.org/cgit/dpkg/dpkg.git/tree/scripts/dpkg-scanpackages.pl#n249 (at least this is my understanding)

Reply:
Yes, it needs to be handled manually